addon/appModules/notepad++/editWindow.py

```diff
@@ -126,7 +128,19 @@ def script_goToFirstOverflowingCharacter(self, gesture):
 	script_goToFirstOverflowingCharacter.category = "Notepad++"
 
 	def script_reportLineInfo(self, gesture):
-		ui.message(self.parent.next.next.firstChild.getChild(2).name) 
+		lineInfo = self.parent.next.next.firstChild.getChild(2).name
+		# Get only the numbers we want from the statusBar.
+		lineInfoExpression = re.compile(r"^Ln\D+(\d\w*)\D+(\d\w*)\D+(\d\w*)\D+(\d\w*)")
```

Comment:
Mind putting this outside the function as a constant? That way it doesn't need recompiled every time the function runs.